modules/015-admission-policy-engine/crds/operation-policy.yaml

```diff
@@ -171,6 +171,16 @@ spec:
                     checkContainerDuplicates:
                       type: boolean
                       description: "Check container names and env variables for duplicates."
+                    replicaLimits:
```

Comment:
Why is the `Limits` in the plural form? Previously, there were many limits, but now it seems like there is only one. Or is it about upper limit and lower limit?

Reply:
Exactly, there are two limits for one **Replicas** parameter